src/utils/components/Consoles/ConsoleStandAlone.tsx

```diff
@@ -4,28 +4,39 @@ import { useParams } from 'react-router-dom-v5-compat';
 import { isHeadlessMode } from '@kubevirt-utils/resources/vm';
 import useVMI from '@kubevirt-utils/resources/vm/hooks/useVMI';
 import { isWindows } from '@kubevirt-utils/resources/vm/utils/operation-system/operationSystem';
+import { Bullseye, Spinner } from '@patternfly/react-core';
+import { useFleetK8sAPIPath } from '@stolostron/multicluster-sdk';
 
 import ErrorAlert from '../ErrorAlert/ErrorAlert';
 
 import { getConsoleBasePath } from './utils/utils';
 import Consoles from './Consoles';
 
 const ConsoleStandAlone: FC = () => {
-  const { name, ns } = useParams<{ name: string; ns: string }>();
-  const { vmi, vmiLoadError } = useVMI(name, ns);
+  const { cluster, name, ns } = useParams<{ cluster?: string; name: string; ns: string }>();
+  const [apiPath, apiPathLoaded] = useFleetK8sAPIPath(cluster);
```

Comment:
What if we create a wrapper hook for `useFleetK8sAPIPath` and pull in the cluster there? Between that and pulling in the cluster in the other hooks that need it we can reduce the places in the code that are being touched.

Between that and a wrapper for `useK8sWatchData` that I mentioned in another comment you can avoid prop drilling the cluster value.